src/main/bluesky/parser.py

```diff
@@ -87,6 +89,6 @@ def write_scene_file(filename):
                 file.write(scenetext)
 
 
-# write_scene_file("scenefile.scn")
+write_scene_file("scenefile.scn")
```

Comment:
I could be wrong but it might be useful to not harcode the naming on these files. But for the scene file this could be the case

Reply:
denk dat scenefile wel handig is gwn